app/controllers/course-admin/code-example-insights-index.ts

```diff
@@ -0,0 +1,27 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import type RouterService from '@ember/routing/router-service';
+import type LanguageModel from 'codecrafters-frontend/models/language';
+import Store from '@ember-data/store';
+import type { ModelType } from 'codecrafters-frontend/routes/course-admin/code-example-insights-index';
+
+export default class CodeExampleInsightsIndexController extends Controller {
+  @service declare router: RouterService;
+  @service declare store: Store;
+
+  declare model: ModelType;
+
+  @action
+  onLanguageChange(language: LanguageModel | null) {
```

Comment:
```suggestion
  onLanguageChange(language: LanguageModel) {
```

Let's not allowing transitioning to a state where a language is not present